interfaces/builtin/basedeclaration.go

```diff
@@ -32,15 +32,103 @@ type: base-declaration
 authority-id: canonical
 series: 16
 revision: 0
-plugs:
-  network: true
 slots:
-  network:
-    allow-installation:
-      slot-snap-type:
-        - core
+  bluetooth-control:
+    deny-auto-connection: true
+  bluez:
+    deny-auto-connection: true
+  bool-file:
+    deny-auto-connection: true
+  camera:
+    deny-auto-connection: true
+  content:
+    allow-auto-connection:
+      plug-publisher-id:
+        - $SLOT_PUBLISHER_ID
+  cups-control:
+    deny-auto-connection: true
+  docker:
+    deny-auto-connection: true
+  docker-support:
+    deny-auto-connection: true
+  firewall-control:
+    deny-auto-connection: true
+  fuse-support:
+    deny-auto-connection: true
+  fwupd:
+    deny-auto-connection: true
+  gpio:
+    deny-auto-connection: true
+  hardware-observe:
+    deny-auto-connection: true
+  hidraw:
+    deny-auto-connection: true
+  home:
+    deny-auto-connection: false
+  kernel-module-control:
+    deny-auto-connection: true
+  libvirt:
+    deny-auto-connection: true
+  locale-control:
+    deny-auto-connection: true
+  location-control:
+    deny-auto-connection: true
+  location-observe:
+    deny-auto-connection: true
+  log-observe:
+    deny-auto-connection: true
+  lxd-support:
+    allow-auto-connection:
+      plug-publisher-id:
+        - canonical
+      plug-snap-id:
+        - J60k4JY0HppjwOjW8dZdYc8obXKxujRu
```

Comment:
the issue is not the checks (the checks will come in the next PR), is that there's no way for you to set the right snap-declaration in the store atm, so we break lxd, the idea is to switch to those once we have snap-declarations in the store (I tried to put a comment about that in the relevant test)